src/script/browser/CheckBrowser.ts

```diff
@@ -96,15 +96,17 @@ const checkBrowser = (): void => {
     console.error("This browser doesn't support cookies to run the Wire app!");
     return;
   }
+  // Skip the mobile browser check for OAuth
   if (isOauth()) {
     return;
   }
-  if (!('RTCPeerConnection' in window)) {
+  const isMobile = window.matchMedia('(any-pointer:coarse)').matches;
```

Comment:
I suggest we enhance the mobile detection logic for better reliability. The current `window.matchMedia('(any-pointer:coarse)')` check has some limitations.

It detects touch input, which could give false positives on touch-enabled laptops and desktops (such as Surface devices, touch-screen Windows laptops, etc.). Some desktop users might incorrectly see the "unsupported device" message.

It might be a good idea to look at other checks, like checking the user agent:

```ts
const isMobileDevice = () => {
    const isTouch = ('ontouchstart' in window) || (navigator.maxTouchPoints > 0);
    const isSmallScreen = window.innerWidth <= 768;
    const isMobileUserAgent = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
        navigator.userAgent
    );
    
    return isTouch && (isSmallScreen || isMobileUserAgent);
}
```